package.json

```diff
@@ -233,6 +238,11 @@
 					],
 					"default": "2022",
 					"description": "%autolispext.configuration.helptargetyear%"
+				},
+				"autolispext.completion.FilterMacOS": {
```

Comment:
what is this fore?

Reply:
This is from the abstraction section above:
<q>Also includes a new setting that will determine whether Mac OS users actually want to have their auto-completions filter out Windows-Only functions or not. I know if I was logged into a Mac and someone asked me to tweak some network script, I wouldn't want to switch machines or virtualize windows just to get access to something that could help me confidently make the changes and tell the user to try it again. Note, this PR doesn't really use this setting, but the next PR will be a lot to review so it made sense to include what I could</q>